logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepository.kt

```diff
@@ -236,8 +236,16 @@ internal class ConnectionDataSource(
                 )
             }
 
+            ACCEPTED -> {
```

Comment:
I think (just minor stuff) but we could add a `TODO` as well, related to be removed, when we find the proper solution

Reply:
I'll add it in next pr, just now don't have access and don't want to delay the merge. Thanks